docs/sources/flow/reference/components/prometheus.integration.redis.md

```diff
@@ -0,0 +1,104 @@
+---
+# NOTE(rfratto): the title below has zero-width spaces injected into it to
+# prevent it from overflowing the sidebar on the rendered site. Be careful when
+# modifying this section to retain the spaces.
+#
+# Ideally, in the future, we can fix the overflow issue with css rather than
+# injecting special characters.
+
+title: prometheus.​integration.redis
+---
+
+# prometheus.integration.redis
+The `prometheus.integration.redis` component embeds
+[redis_exporter](github.com/oliver006/redis_exporter) for collecting metrics from a redis database.
+
+## Usage
+
+```river
+prometheus.integration.redis "LABEL"{
+}
+```
+
+## Arguments
+The following arguments can be used to configure the exporter's behavior.
+Omitted fields take their default values.
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`redis_addr`                  | `string`   | Address (host and port) of the Redis instance we should connect to. This could be a local Redis instance (e.g. `localhost:6379`), or the address of a remote server. | | yes
```

Comment:
```suggestion
`redis_addr`                  | `string`   | Address (host and port) of the Redis instance to connect to. This can be a local Redis instance (e.g., `localhost:6379`) or the address of a remote server. | | yes
```